lib/cassandra/cql.lua

```diff
@@ -156,6 +156,10 @@ local FRAME_FLAGS  = {
   WARNING          = 0x08,
 }
 
+local function get_error_message(code)
+  return ERROR_TRANSLATIONS[code] or 'UNKNOWN'
```

Comment:
you need to return the code somehow, otherwise this won't be useful at all...

Something like: `return ERROR_TRANSLATIONS[code] or string.format('Unknown code: %d', code)`

Reply:
So it looks like the return code is returned by the read_body handler, and optionally returned by `cluster:execute`. After we merge this, we should definitely look into incorporating the return code into what we log